File: algorithms/1_easy/C/03701_compute_alternating_sum/solution.cpp
class Solution {
public:
    int alternatingSum(vector<int>& nums) {
        int sum {0};
        const size_t len{nums.size()};

        for (size_t i{0}; i < len; ++i) {
            if ((i & 1) == 0) {
                sum += nums[i];
            } else {
                sum -= nums[i];
            }
        }

        return sum;
    }
};

File: algorithms/1_easy/C/03701_compute_alternating_sum/solution.rs
impl Solution {
    pub fn alternating_sum(nums: Vec<i32>) -> i32 {
        let mut sum = 0;
        let nums_len = nums.len();

        for i in 0..nums_len {
            if (i & 1) == 0 {
                sum += nums[i];
            } else {
                sum -= nums[i];
            }
        }

        sum
    }
}
